plugins/woocommerce/src/Internal/Admin/Loader.php

```diff
@@ -313,15 +313,12 @@ public static function add_component_settings( $settings ) {
 			$settings['stockStatuses'] = self::get_order_statuses( wc_get_product_stock_status_options() );
 			$settings['currency']      = self::get_currency_settings();
 			$settings['locale']        = [
-				'siteLocale'    => isset( $settings['siteLocale'] )
-					? $settings['siteLocale']
-					: get_locale(),
-				'userLocale'    => isset( $settings['l10n']['userLocale'] )
-					? $settings['l10n']['userLocale']
-					: get_user_locale(),
-				'weekdaysShort' => isset( $settings['l10n']['weekdaysShort'] )
-					? $settings['l10n']['weekdaysShort']
-					: array_values( $wp_locale->weekday_abbrev ),
+				'siteLocale'    => $settings['siteLocale']
+					?? get_locale(),
+				'userLocale'    => $settings['l10n']['userLocale']
+					?? get_user_locale(),
+				'weekdaysShort' => $settings['l10n']['weekdaysShort']
+					?? array_values( $wp_locale->weekday_abbrev ),
```

Comment:
```suggestion
				'siteLocale'    => $settings['siteLocale'] ?? get_locale(),
				'userLocale'    => $settings['l10n']['userLocale'] ?? get_user_locale(),
				'weekdaysShort' => $settings['l10n']['weekdaysShort'] ?? array_values( $wp_locale->weekday_abbrev ),
```

It's a good opportunity to single-line these statements (splitting them across lines is probably harming readability now we are using `??`).